### PR TITLE
User model

### PR DESCRIPTION
**Pull request by luis8alopez**
## Summary

- Adding new user model using sequelize
- New migration created: creation and deletion of a table

## Trello Card

https://trello.com/c/VcVf61fS/21-modelo-de-usuario

### CODE REVIEW

File: app/models/user.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      lastName: {
+        type: DataTypes.STRING,
+        allowNull: false
+      },
+      email: {
+        type: DataTypes.STRING,
+        allowNull: false,
+        unique: true,
+        validate: {
+          is: '^[a-zA-Z]+.[a-zA-Z]+@wolox.+[(co|ar|cl|mx)]$'
```
Comment (shernanm) [inline app/models/user.js:25]: I would think that at the moment this validation is fine but eventually it should be done in the middleware in conjunction with the schemas (there is a card for this).

File: migrations/migrations/20210331154903-users.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable(
+      'User',
```
Comment (dbenavidesv) [inline migrations/migrations/20210331154903-users.js]: table names should be in lowercase and `spine-case`

File: migrations/migrations/20210331154903-users.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+          allowNull: false,
+          unique: true
+        },
+        firstName: {
+          type: Sequelize.STRING,
+          allowNull: false
+        },
+        lastName: {
+          type: Sequelize.STRING,
+          allowNull: false
```
Comment (dbenavidesv) [inline migrations/migrations/20210331154903-users.js]: table fields should be in `snake_case`

File: app/models/user.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define(
+    'User',
+    {
+      id: {
+        type: DataTypes.INTEGER,
+        primaryKey: true,
+        autoIncrement: true
+      },
+      firstName: {
```
Comment (dbenavidesv) [inline app/models/user.js:10]: As the actual fields of the table would be in `snake_case` you need to find a way to map this `camelCase` properties

File: app/models/user.js (hunk 1 above)
Comment (luis8alopez) [reply app/models/user.js:25]: Understood, i'll leave it there, then i'll change it when i get to that card

File: app/models/user.js [hunk 5]
```diff
@@ -0,0 +1,48 @@
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define(
+    'user',
+    {
+      id: {
+        type: DataTypes.INTEGER,
+        primaryKey: true,
+        autoIncrement: true
+      },
+      first_name: {
```
Comment (dbenavidesv) [inline app/models/user.js]: model properties should be in `camelCase` that's why previously I commented that you should find a way to map the `camelCase` properties of the model to the `snake_case` actual fields of the table

**Review by dbenavidesv — APPROVED** [review]

**Review by shernanm — APPROVED** [review]

**Pull request merged**